warehouse/templates/base.html

```diff
@@ -90,8 +90,8 @@
           <li>
             <h3 {{ l20n("getHelp") }}>Get Help</h3>
           </li>
-          <li><a {{ l20n("installingPackages") }} href="//packaging.python.org/en/latest/installing/">Installing Packages</a></li>
-          <li><a {{ l20n("uploadingPackages") }} href="//packaging.python.org/en/latest/distributing/">Uploading Packages</a></li>
+          <li><a {{ l20n("installingPackages") }} href="//packaging.python.org/en/latest/installing/">Installing Projects</a></li>
+          <li><a {{ l20n("uploadingPackages") }} href="//packaging.python.org/en/latest/distributing/">Uploading Projects</a></li>
```

Comment:
I think these are clearer as ``Packages``, this is about installing/uploading files generally, not the PyPI concept of Project.